package.json

```diff
@@ -28,9 +28,9 @@
     "@ethersproject/bignumber": "^5.7.0",
     "@ethersproject/bytes": "^5.7.0",
     "@ethersproject/providers": "^5.7.0",
-    "@metamask/base-controller": "^2.0.0",
-    "@metamask/controller-utils": "^3.3.0",
-    "@metamask/network-controller": "^8.0.0",
+    "@metamask/base-controller": "^3.0.0",
+    "@metamask/controller-utils": "^4.1.0",
+    "@metamask/network-controller": "^10.3.0",
```

Comment:
Just to note for when we create the release: the state of the NetworkController changed in [9.0.0](https://github.com/MetaMask/core/blob/main/packages/network-controller/CHANGELOG.md#900); specifically, the chain ID in the provider config is now a hex string rather than a decimal number. In theory this _could_ impact SmartTransactionsController, since it takes an `onNetworkStateChange` callback which is called with the NetworkController state. That said, I don't believe this would cause a breaking change in practice, as SmartTransactionsController already assumes that the chain ID was at least a string ([proof 1](https://github.com/MetaMask/smart-transactions-controller/blob/ddadfc8e9c6fe0c4272709248e85380cddd0aa1f/src/SmartTransactionsController.ts#L45), [proof 2](https://github.com/MetaMask/smart-transactions-controller/blob/ddadfc8e9c6fe0c4272709248e85380cddd0aa1f/src/SmartTransactionsController.ts#L51)).

Reply:
It's still a state change though, from binary string to hex string. Minimally this will require a state change, making this a breaking change.

I think it would be safer to update the type from `string` to `Hex`  as we bring in v9 of the network controller. Then TypeScript can help us ensure we don't forget to handle this format change.

Reply:
Oh, actually, nevermind. Chain ID is already hex here, just not typed as such.

e.g https://github.com/MetaMask/smart-transactions-controller/blob/ddadfc8e9c6fe0c4272709248e85380cddd0aa1f/src/utils.ts#L26 and https://github.com/MetaMask/smart-transactions-controller/blob/main/src/constants.ts

Nevermind then! The type change would be a nice improvement but I guess it doesn't need to coincide with this update.